src/main/java/winstone/Launcher.java

```diff
@@ -358,6 +359,8 @@ public boolean isRunning() {
      * listener thread. For now, just shut it down with a control-C.
      */
     public static void main(String[] argv) throws IOException {
+        Map<String, String> args = getArgsFromCommandLine(argv);
```

Comment:
Critical that we parse `--logfile` and redirect the streams _before_ initializing JUL, as explained in the block comment in the `Logger` class.